src/arcade/patch/parameter.patch.xml

```diff
@@ -48,6 +48,7 @@
     <population.process process="metabolism" id="CONSTANT_GLUCOSE_UPTAKE_RATE" value="929.88" unit="fmol glucose/min/M glucose" description="constant glucose uptake rate" />
     <population.process process="metabolism" id="CONSTANT_ATP_PRODUCTION_RATE" value="4.9817" unit="fmol ATP/cell/min" description="constant ATP production rate" />
     <population.process process="metabolism" id="CONSTANT_VOLUME_GROWTH_RATE" value="2.819" unit="um^3/min" description="constant volume growth rate"/>
+    <population.process process="metabolism" id="INITIAL_GLUCOSE_CONCENTRATION" value="0" unit="fmol" description="initial cell internal glucose concentration"/>
```

Comment:
Don't love this being 0 by default

Reply:
What default values would you suggest?

Reply:
Isn't this value being 0 the source of the bug that you had identified before?

Reply:
Yeah - it probably makes sense to set it to the average(?) internal glucose amt of engineered cells, Im just not sure what that is.

Reply:
For now Ill set it as 30fmol since thats what CARCADE seems to be giving me, but we can revisit this value

Reply:
@allison-li-1016 @cainja Possible alternative: keep the parameter as a concentration that matches the glucose concentration and do the conversion to mass in the constructor by multiplying by cell volume?